ddtrace/contrib/pytest/plugin.py

```diff
@@ -127,6 +128,8 @@ def pytest_runtest_protocol(item, nextitem):
             span.set_tag(test.SUITE, item.dtest.globs["__name__"])
         span.set_tag(test.TYPE, SpanTypes.TEST.value)
 
+        span.set_tag(test.FRAMEWORK_VERSION, pkg_resources.get_distribution("pytest").version)
```

Comment:
```suggestion
        span.set_tag(test.FRAMEWORK_VERSION, pytest.__version__)
```

`pkg_resources` is really heavy/very expensive dependency to import.

It is probably "ok", but since we can capture this in a better way, better to avoid.

Reply:
oh I didn't know of this way. Yeah, sure, much better. Thanks!